source/MaterialXTest/MaterialXGenMdl/GenMdl.cpp

```diff
@@ -114,53 +114,126 @@ void MdlShaderGeneratorTester::compileSource(const std::vector<mx::FilePath>& so
     if (sourceCodePaths.empty() || sourceCodePaths[0].isEmpty())
         return;
     
-    std::string mdlcExec(MATERIALX_MDLC_EXECUTABLE);
-    if (mdlcExec.empty())
-    {
-        return;
-    }
-
     mx::FilePath moduleToTestPath = sourceCodePaths[0].getParentPath();
     mx::FilePath module = sourceCodePaths[0];
     std::string moduleToTest = module[module.size()-1];
     moduleToTest = moduleToTest.substr(0, moduleToTest.size() - sourceCodePaths[0].getExtension().length() - 1);
 
     mx::StringVec extraModulePaths = mx::splitString(MATERIALX_MDL_MODULE_PATHS, ",");
 
-    std::string mdlcCommand = mdlcExec;
-    for (const std::string& extraPath : extraModulePaths)
+    std::string renderExec(MATERIALX_MDL_RENDER_EXECUTABLE);
```

Comment:
Current MDLC + render tests. I want to migrate so that generation tests don't get run twice (once for gen and once for render but gen has option to run render). For later.